contrib/zenith_test_utils/zenith_test_utils--1.0.sql

```diff
@@ -12,3 +12,9 @@ RETURNS VOID
 AS 'MODULE_PATHNAME', 'clear_buffer_cache'
 LANGUAGE C STRICT
 PARALLEL UNSAFE;
+
+CREATE FUNCTION get_raw_page_at_lsn(text, text, int8, pg_lsn)
+RETURNS bytea
+AS 'MODULE_PATHNAME', 'get_raw_page_at_lsn'
+LANGUAGE C STRICT
+PARALLEL UNSAFE;
```

Comment:
I'm surprised this takes the relation name as 'text', rather than as 'regclass'. We've switched most functions to 'regclass' over the years (see commit 1dc118660b8f12d3bdec94b6f6e1068966ca62e9, for example). I see that's copied from get_raw_page in pageinspect; I'm surprised we haven't changed get_raw_page to use 'regclass'. Maybe consider using regclass here anyway, and maybe propose a patch to upstream to change pageinspect, too.